modules/mauth-authenticator-apachehttp/src/main/java/com/mdsol/mauth/apache/HttpClientPublicKeyProvider.java

```diff
@@ -102,9 +105,7 @@ private class PublicKeyResponseHandler implements ResponseHandler<String> {
     @Override
     public String handleResponse(HttpResponse response) throws IOException {
       if (response.getStatusLine().getStatusCode() == HttpStatus.SC_OK) {
-        if (ttl == null) {
-          ttl = getMaxAge(response).orElse(configuration.getTimeToLive());
-        }
+        ttl = getMaxAge(response).orElse(configuration.getTimeToLive());
```

Comment:
Do the additional writes to `ttl` have an effect if the cache was already setup?

Reply:
See https://github.com/mdsol/mauth-jvm-clients/pull/142/commits/dd0570b356fba0741430abc554e7265a8a7a554b